### ISSUE TITLE
cmark-gfm is now in debian unstable, perhaps remove the provided debian directory from master?

### ISSUE THREAD

**Issue by keith-packard** [github/cmark#122]
I packaged version 0.28.3.gfm.17 for debian unstable and uploaded it a couple of days ago. It's now in the debian unstable repository, available for anyone to use. The repository containing the packaging changes are available in [Debian's gitlab instance](https://salsa.debian.org/keithp/cmark-gfm).

I had to remove the provided debian directory and replace it; it would be nice to either have the debian directory removed from this repository or to have this repository carry the debian packaging information on a separate branch.

**Comment by kivikakk** [comment]
Thanks so much for doing this! I haven't done any Debian packaging in a long while, so I'll take your advice and remove it from the tree here. 👍

**Comment by kivikakk** [comment]
Done in 975a9d1.

**Comment by keith-packard** [comment]
Ashe Connor <notifications@github.com> writes:

> Thanks so much for doing this! I haven't done any Debian packaging in
> a long while, so I'll take your advice and remove it from the tree
> here.

No worries at all; I had actually done this packaging months ago and
didn't manage to put it upstream until I found another use for it this
week. I was surprised to see it land in the archive so quickly though;
there are over 400 packages awaiting review.

I did use your descriptions for the packages, they were way better than
what I'd written.

Let me know if there's something I missed, or some mistake in how things
are packaged that's breaking things.

-- 
-keith
